avsomr/avsimport.cpp

```diff
@@ -81,10 +83,15 @@ static Score::FileError doImportMSMR(MasterScore* score
             if (err != Score::FileError::FILE_NO_ERROR)
                   return err;
 
+            // corrections after import
             Text* title = score->getText(Tid::TITLE);
             if (title)
                   title->setPlainText(baseName);
 
+            score->setMetaTag("movementNumber", "");
+            score->setMetaTag("movementTitle", "");
```

Comment:
Don't these come empty and automatically by default?

Reply:
hmm, for me, the values are set for them.   
I noticed that if cloud recognition, then they are empty, if local, then written. Perhaps different versions of Audiveris are used, I will try to find out later.

Reply:
Clearing them if set is OK IMHO.